WorkloadManagementSystem/Service/SandboxStoreHandler.py

```diff
@@ -244,15 +244,14 @@ def __networkToFile( self, fileHelper, destFileName = False ):
     except:
       pass
     try:
-      fd = open( destFileName, "wb" )
-    except Exception, e:
+      with open( destFileName, "wb" ) as fd:
+        result = fileHelper.networkToDataSink( fd, maxFileSize = self.__maxUploadBytes )
+    except Exception as e:
       return S_ERROR( "Cannot open to write destination file %s" % destFileName )
-    result = fileHelper.networkToDataSink( fd, maxFileSize = self.__maxUploadBytes )
     if not result[ 'OK' ]:
       return result
     if tfd:
```

Comment:
Sorry, I was expressing myself wrongly...

When creating the temporary file name, the file is created and open as "tfd". Then a few lines later it is again open as "fd" and this is what is used for writing into it...

Then the file is closed using its first handle "tfd" and "tfd" is checked as it could be False, right!

What I claim is that currently "tfd" is not used anywhere while it should, so what should be removed IMHO is line 246 as in 247 tfd should be used instead of fd

Hope it is clearer now...